lib/service/connect.go

```diff
@@ -55,8 +56,9 @@ func (process *TeleportProcess) reconnectToAuthService(role teleport.Role) (*Con
 			// if connected and client is present, make sure the connector's
 			// client works, by using call that should succeed at all times
 			if connector.Client != nil {
-				_, err = connector.Client.GetNamespace(defaults.Namespace)
+				pingResponse, err := connector.Client.Ping(context.TODO())
 				if err == nil {
+					process.setAuthServerFeatures(*pingResponse.GetServerFeatures())
```

Comment:
`pingResponse.GetServerFeatures()` can return nil and then this will panic.

Reply:
checked for nil before dereferencing